src/sqlfluff/core/rules/std/L042.py

```diff
@@ -0,0 +1,81 @@
+"""Implementation of Rule L042."""
+
+from sqlfluff.core.rules.base import BaseCrawler, LintResult
+
+
```

Comment:
@alanmcruickshank I think we're missing the `@document_configuration` decorator here. I'll make a quick PR to add it in